solr/core/src/test/org/apache/solr/core/TestConfigSets.java

```diff
@@ -56,16 +56,12 @@ public CoreContainer setupContainer(String configSetsBaseDir) {
   public void testDefaultConfigSetBasePathResolution() {
     Path solrHome = Paths.get("/path/to/solr/home");
 
-    NodeConfig config =
-        SolrXmlConfig.fromString(
-            solrHome, "<solr><str name=\"configSetBaseDir\">configsets</str></solr>");
+    NodeConfig config = SolrXmlConfig.fromString(solrHome);
     assertThat(
         config.getConfigSetBaseDirectory().toAbsolutePath(),
         is(Paths.get("/path/to/solr/home/configsets").toAbsolutePath()));
 
-    NodeConfig absConfig =
-        SolrXmlConfig.fromString(
```

Comment:
I think we still need an XmlConfigFile constructor taking as input an InputStream